test/persistence/mzcompose.py

```diff
@@ -166,3 +167,28 @@ def workflow_compaction(c: Composition) -> None:
     c.rm("materialized", "testdrive-svc", destroy_volumes=True)
 
     c.rm_volumes("mzdata")
+
+
+# Run tests using the S3 implementation of Blob.
+#
+# TODO: the kafka-sources test has a failpoints test that specifically use a file
```

Comment:
how hard is it to separate failpoints out into its own workflow?

Reply:
Philip already did so in another PR that he opened today! I'll wait for that one to merge (and remove the todo) before merging this

Reply:
ha! I was looking at that next and came back here to amend, but you were too fast :)

Reply:
https://github.com/MaterializeInc/materialize/pull/10662 as been merged, so please proceed.